### DISCUSSION TITLE
Please add the "helmholtz-munich.de" to the valid school email domain names for Github Student Benefits.

### DISCUSSION THREAD

**Discussion by mojtababahrami** [community/community#56836] [GitHub Education]
### Hi everyone,

Hi
Github Student Benefits only recognizes the "@helmholtz-muenchen.de" as a valid school email address domain. However, the new students are only getting the "@helmholtz-munich.de" domain and not the other one. Please add this second one to your database.

The original website: https://www.helmholtz-muenchen.de/ is also being redirected to the new https://www.helmholtz-munich.de/!

Thanks in advance

**Comment by DSDmark** [comment]
Hi @mojtababahrami1993,

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **errors** which you were facing 👍.

**Reply by mojtababahrami** [reply]
Hi @DSDmark 
This is what I mean:
When applying for GitHub student benefits it only recognizes the "@helmholtz-muenchen.de" as a valid school-issued email address. 

![image](https://github.com/community/community/assets/58011118/bfca608a-3075-47b0-b6e8-63f5bfde2ace)

**Reply by DSDmark** [reply]
@mojtababahrami1993,

As you can see, it is clearly mentioned for your school **German Research Center for Environmental Heal** only that kind of email valid `helmholtz-muenchen.de`

> We require applicants of **German Research Center for Environmental Health** (Helmholtz
Zentrum München) to use one of these school-issued email addresses to apply: `helmholtz-muenchen.de`

It is your school related issue I say, that, they are you support email which you want. So, you contact the German Research Center for Environmental Health (Helmholtz Zentrum München) directly to inform them about the issue and see if they can assist you in resolving it.

**Reply by mojtababahrami** [reply]
I know it is clearly mentioned :)
I opened this discussion to ask the Github support/maintainer to resolve this issue. Similar to [this one](https://github.com/orgs/community/discussions/51855).
Thanks for your reply by the way.

**Comment by samus-aran** [comment]
Hi @mojtababahrami1993 

We've raised this with the internal team. We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate. We'll update this thread when we have any updates 👍

**Reply by samus-aran** [reply]
Hi @mojtababahrami1993 

We have updated the school details to reflect the change of the domain name. Please re-apply and let us know how you get on 👍

**Reply by mojtababahrami** [reply]
Thank you @samus-aran! It worked.
